src/main/java/com/github/cchacin/cucumber/steps/RedisSteps.java

```diff
@@ -0,0 +1,14 @@
+package com.github.cchacin.cucumber.steps;
+
+import cucumber.api.java.en.Given;
+import redis.clients.jedis.Jedis;
+
+public class RedisSteps {
+    private final Jedis jedis = new Jedis("localhost");
+
+    @Given("^I have cleaned db (\\d+)$")
```

Comment:
The regexp hasn't a redis related name like "I have a clean redis db" or something similar

And also would be great to make "db" optional in the regexp

WDYT